src/http/client.rs

```diff
@@ -26,6 +26,7 @@ use super::{
     HttpError,
     LightMethod,
     MessagePagination,
+    MultipartUpload,
```

Comment:
This should be imported directly along with `super::multipart::Multipart`, a few lines above this.